docs/extra/optimize_function.rst

```diff
@@ -0,0 +1,297 @@
+********************
+Optimizing functions
```

Comment:
I suggest "finding minima" or something like that. When I read it first, I thought this is about optimizing the code to make functions run faster. Maybe I'm too much of a coder an normal users won't think that way.

Reply:
Maybe "minimizing functions"?

Reply:
I note that the scipy docs are "Optimization"

https://docs.scipy.org/doc/scipy/reference/tutorial/optimize.html

and we have a big section on "Optimisers" at https://sherpa.readthedocs.io/en/latest/optimisers/index.html so I'm less concerned about this issue than you are. I don't particularly like the title but I do not have the energy/time to come up with a better one.

I will add a link to the scipy docs to let users know what we're trying to do here.

Reply:
OK, good point. Fine with me.